core/src/main/java/com/linecorp/armeria/server/http/dynamic/Get.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.http.dynamic;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.linecorp.armeria.common.http.HttpMethod;
+
+/**
+ * Class for mapping {@link HttpMethod#GET} HTTP Method onto specific method.
```

Comment:
Global comment:

- Class -> Annotation
- Remove 'HTTP Method' because the `{@link HttpMethod#...}` already tells a user that it's about an HTTP method.